src/plugins/intel_npu/src/utils/src/zero/zero_api.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "intel_npu/utils/zero/zero_api.hpp"
+
+#include "openvino/core/except.hpp"
+#include "openvino/util/file_util.hpp"
+#include "openvino/util/shared_object.hpp"
+
+namespace intel_npu {
+ZeroApi::ZeroApi() {
+    const std::string baseName = "ze_loader";
+    try {
+        auto libpath = ov::util::make_plugin_library_name(ov::util::get_ov_lib_path(), baseName + OV_BUILD_POSTFIX);
+#if defined(OPENVINO_ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
+        this->lib = ov::util::load_shared_object(ov::util::string_to_wstring(libpath).c_str());
+#else
+        this->lib = ov::util::load_shared_object(libpath.c_str());
```

Comment:
Just a curiosity here, did you test these changes on Linux as well?

Reply:
I updated the ticket with the testing scenarios.